### DISCUSSION TITLE
[任务]翻译校对 "Apache Innovation" 中英文字幕

### DISCUSSION THREAD

**Discussion by JesseZhou-1** [alc-beijing/SIG#3] [翻译相关话题]
本翻译任务主要希望将Apache的纪录片——[Apache Innovation](https://www.youtube.com/watch?v=qkvqJaX4S50) 。认领任务请点击[这里](https://github.com/alc-beijing/translation/issues/50)。**请注意，该任务被拆分为9个子任务，大家可以任选一个翻译。**

**Comment by WillemJiang** [comment]
大家可以在此提出想要做的任务，方便协调。
